src/bluecore_api/app/routes/instances.py

```diff
@@ -76,8 +72,8 @@ async def update_instance(
 
     # Update fields if they are provided
     if instance.data is not None:
-        graph = rdflib.Graph().parse(data=instance.data, format="json-ld")
```

Comment:
There's no need to create a graph and frame it here, because it happens automatically when `.data` is assigned to.